<?xml version="1.0" encoding="UTF-8"?>
<testng-results ignored="0" total="6" passed="5" failed="1" skipped="0">
  <reporter-output>
  </reporter-output>
  <suite started-at="2022-08-29T17:34:35Z" name="Default suite" finished-at="2022-08-29T17:34:44Z" duration-ms="9857">
    <groups>
    </groups>
    <test started-at="2022-08-29T17:34:35Z" name="Default test" finished-at="2022-08-29T17:34:44Z" duration-ms="9857">
      <class name="com.pojo.SampleLogin">
        <test-method signature="login()[pri:1, instance:com.pojo.SampleLogin@60129b9a]" started-at="2022-08-29T17:34:35Z" name="login" finished-at="2022-08-29T17:34:39Z" duration-ms="4573" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- login -->
        <test-method signature="addUserAddress()[pri:2, instance:com.pojo.SampleLogin@60129b9a]" started-at="2022-08-29T17:34:39Z" name="addUserAddress" finished-at="2022-08-29T17:34:40Z" duration-ms="1238" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- addUserAddress -->
        <test-method signature="updateUserAddress()[pri:3, instance:com.pojo.SampleLogin@60129b9a]" started-at="2022-08-29T17:34:40Z" name="updateUserAddress" finished-at="2022-08-29T17:34:42Z" duration-ms="1118" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- updateUserAddress -->
        <test-method signature="deleteUserAddress()[pri:4, instance:com.pojo.SampleLogin@60129b9a]" started-at="2022-08-29T17:34:42Z" name="deleteUserAddress" finished-at="2022-08-29T17:34:42Z" duration-ms="907" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- deleteUserAddress -->
        <test-method signature="getUserAddress()[pri:5, instance:com.pojo.SampleLogin@60129b9a]" started-at="2022-08-29T17:34:42Z" name="getUserAddress" finished-at="2022-08-29T17:34:43Z" duration-ms="910" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- getUserAddress -->
        <test-method signature="changeProfilePic()[pri:6, instance:com.pojo.SampleLogin@60129b9a]" started-at="2022-08-29T17:34:43Z" name="changeProfilePic" finished-at="2022-08-29T17:34:44Z" duration-ms="1026" status="FAIL">
          <exception class="java.lang.AssertionError">
            <message>
              <![CDATA[verify status code expected [200] but found [400]]]>
            </message>
            <full-stacktrace>
              <![CDATA[java.lang.AssertionError: verify status code expected [200] but found [400]
at org.testng.Assert.fail(Assert.java:96)
at org.testng.Assert.failNotEquals(Assert.java:776)
at org.testng.Assert.assertEqualsImpl(Assert.java:137)
at org.testng.Assert.assertEquals(Assert.java:118)
at org.testng.Assert.assertEquals(Assert.java:652)
at com.pojo.SampleLogin.changeProfilePic(SampleLogin.java:44)
at java.base/jdk.internal.reflect.DirectMethodHandleAccessor.invoke(DirectMethodHandleAccessor.java:104)
at java.base/java.lang.reflect.Method.invoke(Method.java:577)
at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:124)
at org.testng.internal.Invoker.invokeMethod(Invoker.java:583)
at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:719)
at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:989)
at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:125)
at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:109)
at org.testng.TestRunner.privateRun(TestRunner.java:648)
at org.testng.TestRunner.run(TestRunner.java:505)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:455)
at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:450)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:415)
at org.testng.SuiteRunner.run(SuiteRunner.java:364)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:84)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1208)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1137)
at org.testng.TestNG.runSuites(TestNG.java:1049)
at org.testng.TestNG.run(TestNG.java:1017)
at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:115)
at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:251)
at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:77)
]]>
            </full-stacktrace>
          </exception> <!-- java.lang.AssertionError -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- changeProfilePic -->
      </class> <!-- com.pojo.SampleLogin -->
    </test> <!-- Default test -->
  </suite> <!-- Default suite -->
</testng-results>
